### PR TITLE
Fwmv2 access list theory

### PR DESCRIPTION
**Pull request by stephamon**
New article from #1934 with editor feedback applied

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-b1236369cf/how-to/).

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-518e33efbb/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c8a01a251d/how-to/)

File: content/dedicated-hosting/firewall-manager-v2-access-list-theory-and-best-practices.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+permalink: firewall-manager-v2-access-list-theory-and-best-practices/
+audit_date:
+title: 'Firewall Manager v2 access-list theory and best practices'
+type: product
```
Comment (DonNateR) [inline content/dedicated-hosting/firewall-manager-v2-access-list-theory-and-best-practices.md]: Metadata needs to be fixed

Comment (stephamon) [inline content/dedicated-hosting/firewall-manager-v2-access-list-theory-and-best-practices.md]: Ah, thanks

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f83387a3ca/how-to/)

**Comment by DonNateR** [comment]
LGTM

**Pull request merged**